### PR TITLE
Fix tabs

### PR DESCRIPTION
**Pull request by tmccoy529**
Fixed tab highlighting based on if a type selected is an accela type. Cleaned up code by removing some redundant code and unused commented out code. Fixed the selection of website issues to always push up 400 Washington address as the location and preventing users from entering in a different location.

### CODE REVIEW

File: src/components/FormContainer.jsx [hunk 1]
```diff
@@ -7,33 +7,36 @@ const selectTab = (currentTab, tabList)  => {
...
-
 	return (
 		<div className="bc-citysourced-reporter">
 			<ol className="bc-citysourced-reporter-steps">
-				{ tabList.map((tab,index) => {
+				{ tabList.map((tab,id) => {
 					return (
 						<li
-							key={index}
+							key={id}
```
Comment (martypowell) [inline src/components/FormContainer.jsx]: i believe this should be `key={tab.id}`

Comment (martypowell) [reply src/components/FormContainer.jsx]: if so you won't need to pass id in the map function

File: src/components/FormContainer.jsx [hunk 2]
```diff
@@ -7,33 +7,36 @@ const selectTab = (currentTab, tabList)  => {
...
 const FormContainer = props => {
-
+	
 	const tabList = [
-		{description: props.tabNames.map(name => name.Tab1), key: 'ServiceRequestForm', value: 0, shouldDisableForm: false, requiresLocation: true},
-		{description: props.tabNames.map(name => name.Tab2), key: 'ProvideDetails', value: 1, shouldDisableForm: props.shouldDisableForm, requiresLocation: true},
-		{description: props.tabNames.map(name => name.Tab3), key: 'AdditionalInformation', value: 2, shouldDisableForm: props.shouldDisableForm, requiresLocation: props.requiresLocation},
-		{description: props.tabNames.map(name => name.Tab4), key: 'Blank', value: 3, shouldDisableForm: false, requiresLocation: true},
+		{description: props.tabNames.map(name => name.Tab1), id: 0,  key: 'ServiceRequestForm', shouldDisableForm: false, isAccela: true},
+		{description: props.tabNames.map(name => name.Tab2), id: 1, key: 'ProvideDetails', shouldDisableForm: props.shouldDisableForm, isAccela: true},
```
Comment (martypowell) [inline src/components/FormContainer.jsx]: i think we should probably avoid namign this after a vendor product, in the case that this changes to another vendor product. Maybe something like `isPanelRequired` maybe?

File: src/components/ServiceRequestForm.jsx [hunk 3]
```diff
@@ -310,7 +314,7 @@ const ServiceRequestForm = (props, errors, touched) => {
 					const value = requestType.toLowerCase();
 					const subCategories = getSubCategories(Categories, value ? value : value);
 					setSubCategories(subCategories);
-					if (localProps.values['petType'] !== '') {
+					if (petType !== '') {
```
Comment (martypowell) [inline src/components/ServiceRequestForm.jsx]: this can probably be `if (petType) {`

File: src/components/FormContainer.jsx (hunk 1 above)
Comment (tmccoy529) [reply src/components/FormContainer.jsx]: yes, tab.id is correct. This has been changed

File: src/components/ServiceRequestForm.jsx (hunk 3 above)
Comment (tmccoy529) [reply src/components/ServiceRequestForm.jsx]: yup and the above of requestType =='' has been removed as well.

File: src/components/FormContainer.jsx (hunk 2 above)
Comment (tmccoy529) [reply src/components/FormContainer.jsx]: naughty space messed me up. This has been changed

**Pull request merged**